src/test/java/uk/gov/companieshouse/controller/AppealControllerTest_POST.java

```diff
@@ -215,6 +213,17 @@ void whenExceptionFromService_return500() throws Exception {
             .andExpect(status().isInternalServerError());
     }
 
+    @Test
+    void whenIllnessReasonIncludesRelationship_return422() throws Exception {
+        final String invalidRelationshipAppeal = asJsonString("src/test/resources/data/invalidRelationshipAppeal.json", appeal -> { return appeal; });
```

Comment:
Nothing to do with this file as such but notice we have a validAppeal.json file. Can we add the relationshipToCompany field into these as it's a new field?